src/run.ts

```diff
@@ -137,6 +147,23 @@ async function run_all_locally() {
   run_fe_locally(runner);
 }
 
+async function install_deps_for_services(runner: LabeledProcessRunner) {
+  for (const service of deployedServices) {
+    await runner.run_command_and_output(
+      `Installing Dependencies`,
```

Comment:
Does `Installing Dependencies` need to be in backticks?

Reply:
it does not! will change